### ISSUE TITLE
Should PackageDistro and PackageDistroTools be merged into one directory?

### ISSUE THREAD

**Issue by fingolfin** [gap-system/PackageDistroTools#7]
Some bikeshedding, but while writing docs, and then thinking about how to implement GH Actions, I realized that I always need both repos... so do we really want two?

### Advantages of having two repos:
1. The tools to make the distribution are logically separate from the distribution  (then again: the actual end product of the distribution are the `packages.tar.gz` files produced by `assemble-distro.py`...)
2. Separate history for them
3. Right now, PRs to the one are always about "updating the repo", and the other about "updating tools"; if they were in one repo, we'd somehow have to distiniguish those (and run completely different CI)
4. ... ?

### Advantage of having all in one repo:

1. the scripts are useless without the data, all practical uses will require having both repos checked out in parallel
2. it is easier to find certain things; e.g. the scripts are in one repo, the GH Actions in another, where is the documentation, etc.
3. ...

**Comment by fingolfin** [comment]
After discussion on a similar question ("should perhaps https://github.com/gap-infra/integration be merged with the distribution repository"), I think now i see why having two (resp. three) repos is a good idea: it means that we have only one kind of PRs on the distribution repo. That simplifies somet things quite a bit.

**Comment by fingolfin** [comment]
I think the answer is no for now
